backend/lambda/lambda-returns-blog-posts.py

```diff
@@ -0,0 +1,10 @@
+from lambdaretrievedatalibrary.lambda_return_data import lambda_handler
+import json
+
+event = {}
+context = {}
+table = "blog_posting"
+
+data = lambda_handler(event, context, table)
```

Comment:
Please set the lambda_handler function inside each lambda file, and set the search logic inside a separate function you'll import from the module (instead of lambda handler)